sql/moz-fx-data-shared-prod/mozilla_vpn_derived/active_subscription_ids_live/view.sql

```diff
@@ -7,4 +7,11 @@ SELECT
 FROM
   mozdata.mozilla_vpn.all_subscriptions
 CROSS JOIN
-  UNNEST(GENERATE_DATE_ARRAY(DATE(subscription_start_date), DATE(end_date) - 1)) AS active_date
+  UNNEST(
+    GENERATE_DATE_ARRAY(
+      DATE(subscription_start_date),
+      GREATEST(DATE(subscription_start_date), DATE(end_date) - 1)
```

Comment:
That's intentional, as otherwise subscriptions that start and end on the same UTC date don't have any `active_subscription_ids` records and thus aren't included in `subscription_events` counts.

My thinking is this should be inclusive of `subscription_start_date` and exclusive of `end_date`, with the inclusivity of `subscription_start_date` taking precedence for single-day subscriptions.